src/renderer/components/message/Message.tsx

```diff
@@ -542,6 +542,36 @@ export default function Message(props: {
         </div>
       </div>
     )
+  } else if (message.vcardContact) {
```

Comment:
Isn't vcard an attachment that can still have a text associated with it like webxdc messages?
I don't think it is as special of a message as video chats that get fully custom UI

Reply:
cc @r10s @adbenitez ?

Reply:
Yes, it can have a text as well. Moreover, this is already possible in Android